src/FacadeAccountCreation/FacadeAccountCreation.Core/Services/Organisation/OrganisationService.cs

```diff
@@ -279,4 +279,33 @@ public async Task UpdateOrganisationDetails(
 
         response.EnsureSuccessStatusCode();
     }
+
+    public async Task<List<OrganisationNationModel>?> GetOrganisationNationByExternalIdAsync(Guid organisationExternalId)
+    {
+        var url = $"{OrganisationNationUrl}?organisationId={organisationExternalId}";
+
+        try
+        {
+            _logger.LogInformation("Attempting to fetch the nation for an organisation id {organisationExternalId} from the backend", organisationExternalId);
+
+            var response = await _httpClient.GetAsync(url);
+
+            if (response.StatusCode == HttpStatusCode.NotFound) return null;
+
+            response.EnsureSuccessStatusCode();
+
+            return await response.Content.ReadFromJsonWithEnumsAsync<List<OrganisationNationModel>>();
+        }
+        catch (Exception e)
+        {
+            _logger.LogError(e, "Failed to get Organisation nation for Organisation Id: '{organisationExternalId}'", organisationExternalId);
```

Comment:
SonarQube can be fussy about placeholder being in Pascal case i.e. **O**rganisationExternalId

Reply:
done